Source/cmGlobalFastbuildGenerator.cxx

```diff
@@ -702,8 +702,7 @@ class cmGlobalFastbuildGenerator::Detail::Detection
 		for (std::vector<std::string>::iterator i = compileCmds.begin();
 			i != compileCmds.end(); ++i)
 		{
-			std::string & compileCmd = *i;
-			lg->ExpandRuleVariables(compileCmd, compileObjectVars);
+			lg->ExpandRuleVariables(*i, compileObjectVars);
```

Comment:
The only reason was the re-use/shadowing of the variable name compileCmd that is defined in an outer scope.  I'm happy to keep this in, I'll just change the name so gcc stops whinging.